ecosystem/typescript/sdk/src/providers/indexer.ts

```diff
@@ -37,7 +37,7 @@ export class IndexerClient {
   private async queryIndexer<T>(graphqlQuery: GraphqlQuery): Promise<T> {
     const { data } = await axios.post(this.endpoint, graphqlQuery);
     if (data.errors) {
-      throw new Error(`Indexer data error ${data.errors}`);
+      throw new Error(`Indexer data error ${JSON.stringify(data.errors)}`);
```

Comment:
JSON.stringify can make the error very hard to read as it puts it all on a single line.

Reply:
Can we do something similar to what console.log does by default when you do console.log("Message", error)? It prints the object structure out

Reply:
`data.errors` is an array of objects, doing `console.error("msg",data.errors)` would print `msg [Object Object]`

Reply:
How about something like pretty print: https://www.npmjs.com/package/pretty-print-json

Reply:
I don't think this small thing requires to support a new package - more for us to maintain and more packages for our users to install also.
I'll work on a nicer solution though

Reply:
@movekevin updated with a screenshot